### ISSUE TITLE
Magisk 27.0 issue 

### ISSUE THREAD

**Issue by 4rch00** [HuskyDG/magic_overlayfs#68]
If the module is activated, you will lose your root status

**Comment by PJCreation** [comment]
you can rebuild this module, fork this repo and run github actions. then you flash the module, magisk will be normal.

**Comment by rddelacosta** [comment]
> you can rebuild this module, fork this repo and run github actions. then you flash the module, magisk will be normal. [#64 (comment)](https://github.com/HuskyDG/magic_overlayfs/issues/64#issuecomment-2040553149)

What do you mean by rebuilding this module?

**Comment by Cathgao** [comment]
> you can rebuild this module, fork this repo and run github actions. then you flash the module, magisk will be normal. [#64 (comment)](https://github.com/HuskyDG/magic_overlayfs/issues/64#issuecomment-2040553149)

not the solution. Boot loop and Magisk automatic disabled it after flashing the action release, it certainly something wrong with Magisk 27.0

**Comment by weskerty** [comment]
> If the module is activated, you will lose your root status

Do you know how to uninstall the module?
since magisk does not have access to modules or root

**Comment by Cathgao** [comment]
> > If the module is activated, you will lose your root status
> 
> Do you know how to uninstall the module?
> since magisk does not have access to modules or root

just reflash magisk using TWRP and you will get it back. But all modules you have installed before are lost.

**Comment by weskerty** [comment]
> > > Si el módulo está activado, perderás tu estado de root.
> > 
> > 
> > ¿Sabes cómo desinstalar el módulo? 
> > ya que magisk no tiene acceso a módulos ni a root
> 
> simplemente actualiza magisk usando TWRP y lo recuperarás. Pero todos los módulos que haya instalado antes se perderán.

My phone does not have TWRP, it is NokiaG20.

I solved it by installing magisk26, patching the boot.img with magisk26 and flashing it, the modules appear again. Don't disable it, I'll stick with Magisk26.

**Comment by uragan1987** [comment]
@4rch00 did you fixed it?
how it disable it? i have no root access now, and magisk27 want reinstall

**Comment by Sparkkkkie** [comment]
> @4rch00 did you fixed it? how it disable it? i have no root access now, and magisk27 want reinstall

To uninstall the module boot into twrp, go into file manager, go to data/adb/modules there remove the module, then go back to data/adb and check every folder for the module, remove the module from every folder then reboot to system.

**Comment by uragan1987** [comment]
> > @4rch00 did you fixed it? how it disable it? i have no root access now, and magisk27 want reinstall
> 
> To uninstall the module boot into twrp, go into file manager, go to data/adb/modules there remove the module, then go back to data/adb and check every folder for the module, remove the module from every folder then reboot to system.

Yep, did it with KernelSU, booted KernelSU from Fastboot and removed with X-Plore File Manager my model from data/adb/modules

**Comment by user57752** [comment]
hello, I met the same problem. Who knows how to fix it without reinstall the magisk?

**Comment by I-Flip-Zeros** [comment]
> hello, I met the same problem. Who knows how to fix it without reinstall the magisk?

boot into twrp or whatever custom recovery you use, then delete the module folder

**Comment by nkshwtrst** [comment]
Reboot to safe mode, then back. It will disable all modules, but you'll get root access back

**Comment by 1136535305** [comment]
> Reboot to safe mode, then back. It will disable all modules, but you'll get root access back (Work for Xiaomi, but you can try to do the same, maybe It'll help)

also work for Pixel 3
